### ISSUE TITLE
$arguments array emptied on action on table action

### ISSUE THREAD

**Issue by clinvest** [laravel-filament/filament#10320]
### Package

filament/filament

### Package Version

v3.0.0

### Laravel Version

v10.0.0

### Livewire Version

v3.0.0

### PHP Version

PHP 8.2.0

### Problem description

Working (filled arguments array): Filament\Actions\Action
```
Actions\Action::make('Test arguments')
                    ->arguments(['argument-1' => 'XXX'])
                    ->action(function (array $arguments): array {
                        dd($arguments);
                        return $arguments;
                    })
```

Not working (empty arguments array): Filament\Tables\Actions\Action
```
Tables\Actions\Action::make('Test arguments')
                    ->arguments(['argument-1' => 'XXX'])
                    ->action(function (array $arguments): array {
                        dd($arguments);
                        return $arguments;
                    })
```

### Expected behavior

$arguments array should be filled also on Tables\Actions\Action

### Steps to reproduce

- Install the filament issue project
- Go to http://127.0.0.1:8000/admin/users/1/edit and click on "test arguments", arguments array is filled.
- Go to http://127.0.0.1:8000/admin/users and click in the list on "test arguments", arguments array is empty (same code).

### Reproduction repository

https://github.com/clinvest/filament-issue-arguments/tree/main

**Comment by danharrin** [comment]
Fixed by #10323.

**Comment by monzer15** [comment]
I think Forms\Components\Actions\Action has the same issue
